src/messages.rs

```diff
@@ -40,7 +41,7 @@ pub enum Message {
 /// Messages sent via a direct connection.
 ///
 /// Allows routing to directly send specific messages between nodes.
-#[derive(Debug, RustcEncodable, RustcDecodable)]
+#[derive(RustcEncodable, RustcDecodable)]
```

Comment:
Would it make sense to create some wrapper type `BinaryData(Vec<u8>)` instead, manually implement `Debug` for that one, and then keep the `#[derive(Debug)]` on all the message types?